frontend/src/components/Collapsible/Collapsible.tsx

```diff
@@ -0,0 +1,29 @@
+import React, { useState } from 'react';
+import styles from './collapsible.module.css';
+import { up, down } from "./icons";
+
+type CollapsibleSection = {
+  title: string;
+  children: JSX.Element | JSX.Element[];
+}
+
+const Collapsible = ({ title, children }: CollapsibleSection) => {
+  const [expanded, setExpanded] = useState(false);
+  const icon = expanded ? down : up;
+
+  const changeExpanded = () => {
+    setExpanded(!expanded);
+  }
+
+  return (
+    <div className={styles.collapsible}>
+      <div className={styles.banner} onClick={changeExpanded}>
```

Comment:
Don't have to do this. But maybe since this function is so simple, you could inline it? Just have onClick={( ) => setExpanded(!expanded)}